en/index.md

```diff
@@ -53,6 +53,61 @@ technology terms:
  - space-separated, comma-delimited
  - load balancer (only upper case as part of proper name e.g. Elastic Load Balancer)
 
+## Branding
+
+Consistency in branding is important for a number of reasons, including the protection of trademarks where they apply. 
+
+### Ubuntu
```

Comment:
Not sure what is the best place for it, but can we explicitly state that it's "an Ubuntu [machine/system/release/whatever]", not "a Ubuntu"?